### DISCUSSION TITLE
flask db migrate for 'creat view' / 'drop view'

### DISCUSSION THREAD

**Discussion by learnersbox** [sqlalchemy/alembic#1102] [Usage Questions]
How to integrate CREATE VIEW and DROP View as shown in wiki "[Views](https://github.com/sqlalchemy/sqlalchemy/wiki/Views)" ?

I have added the code in my models.py but it seems flask db migrate unable to recognise the same.

(.venv) d:\MyCodes\flask>flask db migrate -m "base-0.02"
INFO  [alembic.runtime.migration] Context impl SQLiteImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
INFO  [alembic.env] No changes in schema detected.

Any help please ?
Regards

**Comment by zzzeek** [comment]
alembic autogenerate does not detect views.   so you need to edit your migration files directly in order to add new views.
